client/branded/src/global-styles/alert.scss

```
@@ -19,3 +20,11 @@
 .alert-link {
     text-decoration: underline;
 }
+
+.alert.alert-info,
+.alert.alert-warning,
+.alert.alert-danger {
```

Is there a reason we can't just match on `.alert`? E.g. do you specifically want to exclude `.alert-secondary`? I think it should apply to all alerts
```suggestion
.alert {
```

---

Since this doesn't use `:where()`, did you test the specificity problem I described in https://github.com/sourcegraph/sourcegraph/issues/20152#issuecomment-822687474? I'm not sure if it's actually a problem, so if you tested that and it was no issue seems fine, but if not I am relatively sure there would be an issue here. As mentioned I think the easiest thing to do would be to add an alert with a button link to the story.